uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/controller/UniProtKBSearchControllerIT.java

```diff
@@ -1412,6 +1412,18 @@ protected String getFieldValueForValidatedField(String searchField) {
                 case "annotation_score":
                     value = "5";
                     break;
+                case "uniref_cluster_50":
+                    value = "UniRef50_P0001";
+                    break;
+                case "uniref_cluster_90":
+                    value = "UniRef90_P0001";
+                    break;
+                case "uniref_cluster_100":
+                    value = "UniRef100_P0001";
+                    break;
```

Comment:
I think the accession part should have another char, perhaps. 
UniRef50_P0001 -> UniRef50_P00001 ?

It probably makes no difference tho!